libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -48,9 +51,8 @@
 <ng-template #empty>
   <div class="empty">
     <p>
-      Try searching for a topic, for example "<a (click)="searchExample()"
-        >JavaScript</a
-      >".
+      Try searching for a topic, for example "<a role="link"
```

Comment:
Really? Is this actually a link?

cf. https://www.w3.org/WAI/PF/HTML/wiki/RoleAttribute

Reply:
Thanks for the pointer. I removed the role attr